engine-vuex/src/wwtaware.ts

```diff
@@ -34,7 +34,8 @@ import {
   GotoRADecZoomParams,
   LoadTourParams,
   LoadImageCollectionParams,
-  WWTEngineVuexState
+  WWTEngineVuexState,
+  ImagesetInfo
```

Comment:
Another small stylistic point — for lists like these I try to maintain alphabetical ordering. In general, if there's any kind of collection of things in a source file, I like to alphabetize unless there's some other ordering or grouping that should be applied.

Also, trailing comma.